runtime/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/stream/KafkaGrpcFetchHeaderHelper.java

```diff
@@ -14,25 +14,42 @@
  */
 package io.aklivity.zilla.runtime.binding.kafka.grpc.internal.stream;
 
+import static io.aklivity.zilla.runtime.binding.kafka.grpc.internal.types.stream.GrpcType.BASE64;
+import static io.aklivity.zilla.runtime.binding.kafka.grpc.internal.types.stream.GrpcType.TEXT;
+
+import java.util.Arrays;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.function.Consumer;
 
+import org.agrona.MutableDirectBuffer;
+
 import io.aklivity.zilla.runtime.binding.kafka.grpc.config.KafkaGrpcCorrelationConfig;
 import io.aklivity.zilla.runtime.binding.kafka.grpc.internal.types.Array32FW;
 import io.aklivity.zilla.runtime.binding.kafka.grpc.internal.types.KafkaHeaderFW;
 import io.aklivity.zilla.runtime.binding.kafka.grpc.internal.types.KafkaOffsetFW;
 import io.aklivity.zilla.runtime.binding.kafka.grpc.internal.types.OctetsFW;
+import io.aklivity.zilla.runtime.binding.kafka.grpc.internal.types.stream.GrpcMetadataFW;
+import io.aklivity.zilla.runtime.binding.kafka.grpc.internal.types.stream.GrpcType;
 import io.aklivity.zilla.runtime.binding.kafka.grpc.internal.types.stream.KafkaDataExFW;
 import io.aklivity.zilla.runtime.binding.kafka.grpc.internal.types.stream.KafkaMergedFetchDataExFW;
 
 public final class KafkaGrpcFetchHeaderHelper
 {
+    private static final int META_PREFIX_LENGTH = 5;
+    private static final byte[] HEADER_META_PREFIX = new byte[5];
+    private static final byte[] META_PREFIX = "meta:".getBytes();
+    private static final byte[] HEADER_BIN_SUFFIX = new byte[4];
+    private static final byte[] BIN_SUFFIX = "-bin".getBytes();
+
```

Comment:
Some of these are constant and readonly, while others are mutable byte arrays populated at runtime while processing messages.

The mutable byte arrays should not be static as those are shared across cores causing corruption of the byte array values due to race conditions.

Please move the mutable byte arrays to non-static instance fields and change the naming style accordingly.